### DISCUSSION TITLE
Regarding the title bar.

### DISCUSSION THREAD

**Discussion by fade2gray** [dnGrep/dnGrep#763] [General]
I once had a deep path and long regex that had the best part of 400 characters in total (the regex was 238), and almost filled the full length (well, 2/3 or more) of the title bar when maximised at 1920x1080 resolution.

Just saying. 😜

**Comment by doug24** [comment]
Sounds to me like you are saying: don't put the search or path in the title bar.  Nothing I can do about how long your search or path is, other than truncate it from the left or right, and there's probably no right answer for everyone.

For me, the reason to do this is so when you have multiple instances of dnGrep running, you can hover the cursor over the Windows task bar and find the right window to bring to the front.  I find that useful.

**Reply by fade2gray** [reply]
Fair point.
